bagherilab/ARCADE
# Add setup file documentation

Resolves #126 

## Summary of changes

- Add documentation for setup file structure for `patch` and `potts` implementations including:
    - sets and series
    - cell populations
    - agent actions
    - lattice layers
    - environment components
- Add "not yet implemented" warnings for `potts` for actions/layers/components
- Example setup file snippets for each section 
- Bumps some dependency versions (to close out #104, #105, and #106)

## Steps to verify

1. Navigate to the `docs` folder
2. Edit the `_config.yml` file to replace `baseurl: ARCADE` wtih `baseurl: ""`
3. Run `docker run -p 4000:4000 -v $(pwd):/site bretfisher/jekyll-serve`
4. Navigate to `http://localhost:4000/setup`

I would appreciate a detailed read through of the contents to make sure everything is clear. I'm planning to do separate PRs for: detailed parameter lists, tutorials with full examples of setup files, methods details. The scope of these changes are specifically on the structure of the setup file, but if you have suggestions on things to add to the docs, please let me know!
